src/igvfd/audit/formatter.py

```diff
@@ -30,3 +31,16 @@ def get_audit_message(audit_function, index=0):
             return json.loads(docstring)[index]
         except:
             return ValueError(f'Docstring: {docstring!r} in function: {audit_function.__name__!r} is not valid JSON format.')
+
+
+def check_list_similiary(list_of_lists: list):
```

Comment:
instead of this function you could do something like:
check_all_same = all(set(sublist) == set(list_of_lists[0]) for sublist in list_of_lists)